src/Sarif/Schemata/sarif-schema.json

```diff
@@ -932,18 +932,13 @@
           "type": "string"
         },
 
-        "messageId": {
-          "description": "The resource id for a plain text message string.",
-          "type": "string"
-        },
-
-        "richText": {
-          "description": "A rich text message string.",
+        "markdown": {
+          "description": "A markdown message string.",
```

Comment:
>markdown [](start = 28, length = 8)

"Markdown" is capitalized; see ["Daring Fireball - Markdown Syntax"](https://daringfireball.net/projects/markdown/syntax) and ["GitHub Flavored Markdown Spec"](https://github.github.com/gfm/).